### DISCUSSION TITLE
Available for pfSense Plus 23.05?

### DISCUSSION THREAD

**Discussion by jchessma** [jaredhendrickson13/pfsense-api#350] [General]
Apologies if this is addressed elsewhere. Are there plans to release a version for pfSense Plus 23.05?

Thanks!

**Comment by jaredhendrickson13** [comment]
Yes it will be included in v1.6.0. There is a development build with support for 23.05 in https://github.com/jaredhendrickson13/pfsense-api/releases/tag/v1.6.0_deved94c44 if you're interested in trying it out. Just note it's still in a pre-release stage right now. 

There are some known issues related to the pfSense plus licensing/certificates that may come up during install. You can refer to #334 for more info on that.

**Comment by jchessma** [comment]
Awesome! Thanks!

**Comment by jwfox5150** [comment]
> Yes it will be included in v1.6.0. There is a development build with support for 23.05 in https://github.com/jaredhendrickson13/pfsense-api/releases/tag/v1.6.0_deved94c44 if you're interested in trying it out. Just note it's still in a pre-release stage right now.
> 
> There are some known issues related to the pfSense plus licensing/certificates that may come up during install. You can refer to #334 for more info on that.

Silly question, but how about support for 23.05.1? Is that included in v1.6.1?

**Reply by jaredhendrickson13** [reply]
@jwfox5150 Yes, both v1.6.0 and v1.6.1 will work for pfSense Plus 23.05.1. In previous releases there was a file that needed to be overridden in order for the package to work; which required a new build was required for each new version. v1.6.0 removed the need to override that file on pfSense CE 2.7.0+ and pfSense Plus 23.05+.
